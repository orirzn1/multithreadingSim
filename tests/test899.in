L9
S8
O7
N6

T0
I@0x00000000
STORE $5, $6, 0x3
SUBI $4, $6, 0x3
SUBI $2, $1, 0x1
ADDI $6, $0, 0x3
ADD $7, $3, $6
ADDI $6, $0, 0x4
ADD $6, $7, $3
LOAD $4, $7, 0x2
ADDI $1, $0, 0x2
LOAD $4, $2, $2
HALT $0

T1
I@0x00000000
LOAD $0, $3, 0x4
ADDI $7, $5, 0x4
SUB $4, $3, $7
SUBI $3, $3, 0x4
ADD $2, $4, $0
ADD $1, $0, $7
HALT $0

T2
I@0x00000000
STORE $7, $1, $6
SUB $4, $1, $6
LOAD $0, $4, $1
ADD $7, $0, $6
SUB $3, $1, $0
ADD $2, $2, $2
ADD $3, $0, $5
ADDI $5, $3, 0x1
ADDI $6, $6, 0x1
HALT $0

T3
I@0x00000000
ADDI $1, $5, 0x2
ADD $0, $1, $7
SUBI $0, $5, 0x3
SUBI $7, $2, 0x4
ADDI $6, $2, 0x4
SUBI $2, $7, 0x3
HALT $0

T4
I@0x00000000
ADDI $4, $0, 0x2
SUBI $2, $3, 0x4
ADD $4, $3, $5
SUBI $0, $6, 0x3
SUB $0, $3, $2
SUB $2, $6, $7
STORE $2, $7, 0x1
ADD $5, $0, $7
HALT $0

T5
I@0x00000000
SUBI $0, $6, 0x2
ADDI $1, $4, 0x1
STORE $6, $2, 0x2
LOAD $1, $7, $7
ADD $5, $3, $0
LOAD $4, $6, 0x4
HALT $0

D@0x0
0xA
0x9
0x6
0xA
0x4
0x5
0xA
0x8
