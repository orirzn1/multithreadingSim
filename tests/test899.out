
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x2	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x3	R7 = 0x3
Register file thread id 1:
	R0 = 0x9	R1 = 0xD	R2 = 0x5	R3 = 0xFFFFFFFC	R4 = 0xFFFFFFFC	R5 = 0x0	R6 = 0x0	R7 = 0x4
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x1	R6 = 0x1	R7 = 0x0
Register file thread id 3:
	R0 = 0xFFFFFFFD	R1 = 0x2	R2 = 0xFFFFFFF9	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x4	R7 = 0xFFFFFFFC
Register file thread id 4:
	R0 = 0x4	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0xFFFFFFFE	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x9	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 3.078431

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x2	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x3	R7 = 0x3
Register file thread id 1:
	R0 = 0x9	R1 = 0xD	R2 = 0x5	R3 = 0xFFFFFFFC	R4 = 0xFFFFFFFC	R5 = 0x0	R6 = 0x0	R7 = 0x4
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x1	R6 = 0x1	R7 = 0x0
Register file thread id 3:
	R0 = 0xFFFFFFFD	R1 = 0x2	R2 = 0xFFFFFFF9	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x4	R7 = 0xFFFFFFFC
Register file thread id 4:
	R0 = 0x4	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0xFFFFFFFE	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x9	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.176471

